CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/MergeNodesByPrefixNGTest.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
Please configure the header template which includes the copyright and github username - see https://github.com/constellation-app/constellation/blob/master/CONTRIBUTING.md#pull-requests